lib/install/management/configure.go

```diff
@@ -240,7 +254,11 @@ func (d *Dispatcher) update(conf *config.VirtualContainerHostConfigSpec, setting
 		}
 	}
 
-	if err = d.reconfigVCH(conf, fmt.Sprintf("[%s] %s/%s", conf.ImageStores[0].Host, d.vmPathName, settings.ApplianceISO)); err != nil {
+	isoFile := settings.ApplianceISO
```

Comment:
Here, if `settings.ApplianceISO == ""`, `isoFile = ""` as well.

```go
        isoFile := ""
 	if settings.ApplianceISO != "" {
 		isoFile = fmt.Sprintf("[%s] %s/%s", conf.ImageStores[0].Host, d.vmPathName, settings.ApplianceISO)
 	}
```

The above snippet is easier to understand and has the same behavior as the original code.